setup.py

```diff
@@ -24,7 +24,7 @@
     author_email='guillaume.camera@people-doc.com, '
                  'guillaume.gerard@people-doc.com',
     install_requires=[
-        'Django<1.10',
+        'Django',
```

Comment:
I would have set `Django<1.11` here instead. Because, it'll install the very latest version of Django here if it's unpinned. What do you think?